hw_1/main_3.py

```diff
@@ -0,0 +1,17 @@
+"""
+Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. 
+Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. 
+Вам требуется написать программу, которая проверяет счастливость билета.
+"""
+num = str(input())
```

Comment:
Функция ввода данных уже возвращает целый строковый тип.
```suggestion
num = input()
```